marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/data-services/mastering/suite-setup.xqy

```diff
@@ -0,0 +1,53 @@
+(: set up temporal fields, range indexes, axes and collection for validateMatchingStep test :)
```

Comment:
I am thinking that due to the complexity of the setup, it would be better to make a data-services/mastering/validateMatchingStep directory so that this setup can be specific to this test. The directory may only have one test module in it, but I think that's fine.

Reply:
Also, I don't know if this setup is done elsewhere in ml-unit-test, but this will likely be useful in data-hub-test-helper.xqy - e.g. "setup-temporal-resources()" and "teardown-temporal-resources()". That's under the assumption that an individual test doesn't need any variance in the temporal resources, it just needs some temporal resources setup.

Reply:
I've broken out the test to a separate directory as suggested. Will push shortly. 

Temporal setup requires multiple transactions. The fields have to exist before the range indexes can be created, and so on. It would be convenient to have temporal helper functions, but it would require multiple separate-transaction evals, or some other way to break it into multiple transactions. That's why I punted, for now. This is the first temporal setup code to be added to the project, so it can be used for reference, or parameterized when we have a second need for it.